### DISCUSSION TITLE
`<code class="notranslate">` replaces first backquote of a task

### DISCUSSION THREAD

**Discussion by ffissore** [community/community#15977] [Projects and Issues] [answered]
When a task of an issue contains `code` and it's turned into a sub-issue, the first backquote is replaced with `<code class="notranslate">`, something like
```
- Rename <code class="notranslate">this` to <code class="notranslate">that`
```

**Comment by emmadesilva** [comment]
Happened to me too, same workflow, creating an issue from a task in https://github.com/hydephp/framework/issues/310

**Comment by ffissore** [comment, marked as answer]
This has just been fixed

**Reply by mjacobus** [reply]
It has, thank you for reporting it!

**Comment by mjacobus** [comment]
A fix has just been deployed. Thank you for reporting this bug ❤️
